src/connect.js

```diff
@@ -20,7 +20,17 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  let obj = {};
+  if (params !== undefined) {
+    obj = params;
```

Comment:
better to write like this
```suggestion
    const obj = params? { ...params } : {};
```